build_request_test.go

```diff
@@ -115,3 +115,28 @@ func TestRequestedAuthnContextIncluded(t *testing.T) {
 	require.Equal(t, el.Tag, "AuthnContextClassRef")
 	require.Equal(t, el.Text(), AuthnContextPasswordProtectedTransport)
 }
+
+func TestScopingIDProviderIncluded(t *testing.T) {
```

Comment:
Worth explicitly asserting the cases when only one is set?

Reply:
Sorry, I don't understand the comment.

Reply:
It looks like this tests when both `ScopingIDPProviderId` and `ScopingIDPProviderName` are set, but not what happens when only one is set. I'm wondering if an explicit test to define the behaviour in that case would add clarity.

Reply:
@lieut-data - Ah,  got it.  Added tests for Omitting ScopingIdProvider.